CHANGELOG.md

```diff
@@ -22,6 +22,7 @@ Ref: http://keepachangelog.com/en/0.3.0/
 
 #### [v4.0.0-beta.3]
 - Add PointCloudLayer
+- FIX: `onHover` and `onClick` are no longer called on layers that are not affected
```

Comment:
Updating the changelog is good but we really need to update the `What's New` and `Upgrade Guide` docs as well. And review the `Layer` and `Layer Lifecycle` docs that mention these props.